tests/unit/testsuite/Bolt/Boltpay/Block/Checkout/BoltpayTest.php

```diff
@@ -245,4 +254,368 @@ public function testGetSaveOrderURL()
 
         $this->assertEquals($expected, $result);
     }
-}
\ No newline at end of file
+
+    /**
+     * @test
```

Comment:
I am definitely not a fan of breaking from universally established long-running naming conventions for test and using annotations, but, it is good to use all the resources available and perhaps we can change to this proposed convention.

Reply:
@mccottry annotations allow use cases in one test.
Also, it is clear what method is covered by this test.
I think five tests named something like
"testIsCustomerGroupDisabledWithEmptyCustomerAndEmptyGroups" is not so clear as simple case
Using "@group" allow to run not all tests but some range.

Reply:
The idea is following existing convention.  If all test methods start with test, and then you decide instead to use a different type of name like `getConfigSelectors` as a new test and use instead the annotation @test, then you have introduced unnecessary disharmony.  A casual reader might interpret `getConfigSelectors` as a local getter method and not a test.  In general, honoring existing convention is encouraged in software engineering in team environments.

However, as I mentioned, we should be open to adopting new and better conventions when easy to adopt.